api-spec/v1/openapi.yaml

```diff
@@ -145,10 +178,43 @@ components:
         - operationId
         - correlationId
         - amount
+    RefundRedirectRequest:
+      type: object
+      properties:
+        transactionId:
+          type: string
+          description: The id of the transaction
+          minLength: 32
+          maxLength: 32
+        touchpoint:
+          type: string
+          description: Name of touchpoint
+        pspId:
+          type: string
+          description: The id of the psp
+        pspTransactionId:
+          type: string
+          description: The id of the psp transaction
+        paymentMethodName:
+          type: string
+          description: The name of the payment method
+    RefundRedirectResponse:
```

Comment:
RefundRedirectResponse is similar to https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/blob/ad647b12aab776e07ea3d98231435e64a4f07bd1/api-spec/client/openapi/redirect/redirect-api.yaml#L128

Reply:
so `transactionId` and `outcome`